### ISSUE TITLE
k8s SaveOutput step returned values

### ISSUE THREAD

**Issue by johnbradley** [Duke-GCB/lando#138]
The SaveOutput step uploads the results of running a workflow to an external data store.
We store two values in bespin-api that are generated by this step: 
- DukeDS project ID of the newly created project
- DukeDS file ID of the README file uploaded to the project
The non-k8s version of lando worker posts this data into the rabbitmq queue.
Then the lando service reads the queue and stores these data in bespin-api.

After #137 is implemented sending this data directly back to lando from this step will not be possible.
Instead we need to store this data in a location where lando can fetch it once it is notified of a changed pod.

Met with @dleehr about this.
Decided to write this data to a file inside the container and have a secondary sidecar container that will cat this file and lando can read the logs from this specially labeled container.  
[Documentation on using a sidecar container.](https://kubernetes.io/docs/concepts/cluster-administration/logging/#streaming-sidecar-container)

Care will need to be taken that the file has been flushed and is complete when it is cat'ed.

**Comment by johnbradley** [comment]
Duplicate of #142
